Makefile

```diff
@@ -164,13 +164,28 @@ define docker-up
 	fi
 endef
 
+docker: docker-
```

Comment:
I was actually just trying to make everything compatible with the prior README, which included the `make docker` step that no longer worked. With just `make lantern` we can eliminate that step, however.